### PR TITLE
Fixes #2118 sum formula for volume plasma

### LINKED ISSUE #2118: Define VolumePlasma as SumFormula
see https://github.com/Open-Systems-Pharmacology/Forum/discussions/1174#discussioncomment-2371523

@Yuri05 Do you know why it's still a hard coded formula?
How could we defined it dynamically? Using Plasma as a tag I guess? Thoughts?

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Dump/PKSimDB_dump.txt [hunk 1]
```diff
@@ -12534,8 +12540,8 @@ INSERT INTO tab_container_parameters VALUES(4701,'GENERAL','MoleculeProperties',
 INSERT INTO tab_container_parameters VALUES(4711,'GENERAL','MoleculeProperties','Sum of active process rates lumen to mucosa',0,1,0,0,'UNDEFINED',1,'LOCAL','SIMULATION',0,0);
 INSERT INTO tab_container_parameters VALUES(4714,'GENERAL','MoleculeProperties','Sum of active process rates lumen to mucosa',0,1,0,0,'UNDEFINED',1,'LOCAL','SIMULATION',0,0);
 INSERT INTO tab_container_parameters VALUES(157,'ORGAN','Kidney','eGFR',0,1,1,0,'GFR',99,'LOCAL','INDIVIDUAL',0,1);
-INSERT INTO tab_container_parameters VALUES(5015,'GENERAL','PROTEIN','Disease factor',1,0,1,0,'DISEASE_STATES',1,'GLOBAL','INDIVIDUAL',0,0);
-INSERT INTO tab_container_parameters VALUES(5941,'GENERAL','TRANSPORTER','Disease factor',1,0,1,0,'DISEASE_STATES',1,'GLOBAL','INDIVIDUAL',0,0);
+INSERT INTO tab_container_parameters VALUES(5015,'GENERAL','PROTEIN','Disease factor',1,0,1,0,'DISEASE_STATE_FACTORS',1,'GLOBAL','INDIVIDUAL',0,0);
+INSERT INTO tab_container_parameters VALUES(5941,'GENERAL','TRANSPORTER','Disease factor',1,0,1,0,'DISEASE_STATE_FACTORS',1,'GLOBAL','INDIVIDUAL',0,0);
```
Comment (msevestre) [inline src/Db/Dump/PKSimDB_dump.txt:12544]: I also created this group here so that those factors are only visible in advanced mode

Comment (msevestre) [reply src/Db/Dump/PKSimDB_dump.txt:12544]: image

Comment (Yuri05) [reply src/Db/Dump/PKSimDB_dump.txt:12544]: @msevestre The PR looks good for me!

Has nothing to do with the PR, but where you are changing the DB right now: can you change **IN DEVELOPMENT** to **ß version** in the description of Cat & Cattle? E.g. `Cat (ß version)`
grafik

File: src/Db/Diff/PKSimDB_diff.txt [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+DELETE FROM tab_rate_container_parameters WHERE calculation_method='Individual_PKSim' AND formula_rate='PARAM_VolumePlasma' AND container_id=4164 AND container_type='COMPARTMENT' AND container_name='Plasma' AND parameter_name='Volume';
+DELETE FROM tab_rate_container_parameters WHERE calculation_method='Individual_PKSim' AND formula_rate='PARAM_VolumePlasma' AND container_id=4187 AND container_type='COMPARTMENT' AND container_name='Plasma' AND parameter_name='Volume';
+DELETE FROM tab_rate_container_parameters WHERE calculation_method='Individual_PKSim' AND formula_rate='PARAM_VolumePlasma' AND container_id=5563 AND container_type='COMPARTMENT' AND container_name='Plasma' AND parameter_name='Volume';
+DELETE FROM tab_rate_container_parameters WHERE calculation_method='Individual_PKSim' AND formula_rate='PARAM_VolumePlasma' AND container_id=5568 AND container_type='COMPARTMENT' AND container_name='Plasma' AND parameter_name='Volume';
+UPDATE tab_species SET display_name='Cat (beta version)' WHERE species='Cat';
+UPDATE tab_species SET display_name='Cattle (IV only) (beta version)' WHERE species='Cattle';
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:44]: @Yuri05 Done

Comment (Yuri05) [reply src/Db/Diff/PKSimDB_diff.txt:44]: Cool :)

**Pull request merged**